src/components-v2/InfluenceVault/InfluenceVaultDetail.tsx

```diff
@@ -53,6 +53,8 @@ const InfluenceVaultDetail = (): JSX.Element => {
 
   const influenceVault = vault ? getInfluenceVaultConfig(vault?.vaultToken) : undefined;
   const classes = useStyles();
+
+  // hmm, wtf dis?
```

Comment:
copied over from refactor pretty sure, doesn't seem to actually do anything probably can just be removed completly